src/proposals/20230816/SparkGoerli_20230816.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { SparkPayloadGoerli, IEngine, Rates, EngineFlags } from '../../SparkPayloadGoerli.sol';
+
+import { IPoolAddressesProvider } from 'aave-v3-core/contracts/interfaces/IPoolAddressesProvider.sol';
+
+/**
+ * @title  August 18, 2023 Spark Goerli Proposal - Update to Aave 3.0.2, Unfreeze sDAI Market, Update DAI Interest Rate Strategy
+ * @author Phoenix Labs
+ * @dev    This proposal upgrades the pool contract implementation to Aave 3.0.2
+ * Forum:  https://forum.makerdao.com/t/phoenix-labs-proposed-changes-for-spark-for-august-18th-spell/21612
+ * Vote:   N/A
+ */
+contract SparkGoerli_20230816 is SparkPayloadGoerli {
+
+    address public constant POOL_ADDRESS_PROVIDER      = 0x026a5B6114431d8F3eF2fA0E1B2EDdDccA9c540E;
+    address public constant POOL_IMPLEMENTATION        = 0xe7EA57b22D5F496BF9Ca50a7830547b704Ecb91F;
+    address public constant DAI                        = 0x11fE4B6AE13d2a6055C8D9cF65c55bac32B5d844;
+    address public constant sDAI                       = 0xD8134205b0328F5676aaeFb3B2a0DC15f4029d8C;
+    address public constant DAI_INTEREST_RATE_STRATEGY = 0x70659BcA22A2a8BB324A526a8BB919185d3ecEBC;
```

Comment:
To validate the source code on Goerli:

1. Installed `sourcify` locally as instructed in the README
2. Edited `sourcify` env config to use valid Goerli and Mainnet nodes
3. Ran `sourcify` server
4. Validated source code through `sourcify` by running:
    ```bash
    ./validate-deploy.sh script/output/5/broadcast/primary-pool-20230714.json
    ```

Results:
```
Submitting verification for [BorrowLogic] "0x512AFEDCF6696d9707dCFECD4bdc73e9902e3c6A".
Contract source code already verified. Storage Timestamp: 2023-08-15T17:43:37.517Z
Start verifying contract `0x9d0053198fc830a68165d85d4b3d33d7c002b7fc` deployed on goerli

Submitting verification for [BridgeLogic] "0x9D0053198fc830a68165d85D4b3D33D7C002B7FC".
Contract source code already verified. Storage Timestamp: 2023-08-15T17:43:38.833Z
Start verifying contract `0xe25f940d74094dc0397f22f6ad6ae656f4e6297b` deployed on goerli

Submitting verification for [EModeLogic] "0xE25f940D74094dc0397F22f6aD6AE656f4e6297b".
Contract source code already verified. Storage Timestamp: 2023-08-15T17:43:40.323Z
Start verifying contract `0xb5dad3aea0d92417f100655d1e9b31fab85304a3` deployed on goerli

Submitting verification for [FlashLoanLogic] "0xb5DaD3AeA0D92417f100655d1e9B31fab85304A3".
Contract source code already verified. Storage Timestamp: 2023-08-15T17:43:42.287Z
Start verifying contract `0x83512f8c629da31b430f5bf68e8ee97322b90c60` deployed on goerli

Submitting verification for [LiquidationLogic] "0x83512f8C629dA31B430F5bF68e8Ee97322b90c60".
Contract source code already verified. Storage Timestamp: 2023-08-15T17:43:44.463Z
Start verifying contract `0x8e343af9223733d2d41f0e66e7600033f0f996e6` deployed on goerli

Submitting verification for [PoolLogic] "0x8E343aF9223733d2d41F0e66E7600033F0f996E6".
Contract source code already verified. Storage Timestamp: 2023-08-15T17:43:45.793Z
Start verifying contract `0x6264de350020ec942213c7742075096680a3a694` deployed on goerli

Submitting verification for [SupplyLogic] "0x6264dE350020EC942213C7742075096680a3A694".
Contract source code already verified. Storage Timestamp: 2023-08-15T17:43:47.250Z

///////////////////////////////// NEW POOL IMPLEMENTATION VERIFICATION //////////////////////////////
Start verifying contract `0xe7ea57b22d5f496bf9ca50a7830547b704ecb91f` deployed on goerli

Submitting verification for [Pool] "0xe7EA57b22D5F496BF9Ca50a7830547b704Ecb91F".
Contract successfully verified
```